packages/protocol/migrationsConfig.js

```diff
@@ -30,6 +30,10 @@ const defaultConfig = {
     minDeposit: 1, // 1 cGLD
     queueExpiry: 7 * 24 * 60 * 60, // 1 week
     referendumStageDuration: 15 * 60, // 15 minutes
+    participationBaseline: 8 / 10,
```

Comment:
How important is it to configure the start value for this vs. letting it evolve organically (i.e. start at 0)

Reply:
Letting it start at 0 could be initially risky. Tezos initializes its baseline at 80%, and I think the effect of this choice won't stay relevant for long. If our belief is that we expect high participation at launch, a reasonable choice may be to pick a value close to our expected participation but lowball to mitigate the chance of temporary deadlock.